### ISSUE TITLE
argtypes definitions with one single argument are not raising a TypeError if not passed as a tuple or list

### ISSUE THREAD

**Issue by s-m-e** [pleiszenburg/zugbruecke#21]
[See here.](https://github.com/pleiszenburg/zugbruecke/blob/20f2e44b57e1a59087eae36683582a9657f1d8f4/src/zugbruecke/core/routine_client.py#L151)

**Comment by s-m-e** [comment]
This is how `ctypes` expects the definition of a single argument:
```
>>> import ctypes 
>>> dll = ctypes.cdll.msvcrt
>>> demo = dll.sqrt
>>> demo.argtypes = (ctypes.c_double,)
>>> demo.restype = ctypes.c_double
>>> demo(9.0)
3.0
```

This is how it throws an error if the `argtypes` are not initialized with a tuple or list:
```
>>> import ctypes
>>> dll = ctypes.cdll.msvcrt
>>> demo = dll.sqrt
>>> demo.argtypes = ctypes.c_double   
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: _argtypes_ must be a sequence of types
```

`zugbruecke` currently crashes when parsing `demo.argtypes = ctypes.c_double`, which is sort of right - it should actually throw a `TypeError` instead (#8).

**Comment by s-m-e** [comment]
Fixed in 679b383.

**Comment by s-m-e** [comment]
Fix released as part of v0.0.8.
